e2e/verify/verify.go

```diff
@@ -289,6 +289,66 @@ func (c *ctx) singularityVerifySigner(t *testing.T) {
 	}
 }
 
+func (c *ctx) checkGroupidOption(t *testing.T) {
+	/*
+		// Create a temp directory where we can store a valid image
+		tempDir, err := ioutil.TempDir("", "")
+		if err != nil {
+			t.Fatalf("failed to create a temporary directory: %s", err)
+		}
+		defer func() {
+			err := os.RemoveAll(tempDir)
+			if err != nil {
+				t.Fatalf("failed to delete temporary directory: %s", err)
+			}
+		}()
+
+		imgPath := filepath.Join(tempDir, "imgTest.sif")
+
+		// Pull a valid image
+		e2e.PullImage(t, c.env, successURL, imgPath)
+	*/
+	cmdArgs := []string{"--groupid", "0", c.successImage}
```

Comment:
If we're going to use `c.successImage` we should use `e2e.PullImage()` for each test, or in `RunE2ETests()` before the tests start to ensure that the image presence isn't dependent on test order. Currently the `singularityVerifyKeyNum()` and `singularityVerifyKeySigner()` call `e2e.PullImage()` within the tests, so that should be changed if you go with the later option